security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,39 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-4-1"]
+== OpenShift Compliance Operator 1.4.1
+
+The following advisory is available for the OpenShift Compliance Operator 1.4.1:
+
+* link:https://access.redhat.com/errata/RHBA-2024:1830[RHBA-2024:1830 - OpenShift Compliance Operator bug fix and enhancement update]
+
+[id="compliance-operator-1-4-1-new-features-and-enhancements"]
+=== New features and enhancements
+
+* With this update, Compliance Operator now provides `OCP4 STIG ID` and `SRG` with the profile rules. (link:https://issues.redhat.com/browse/CMP-2401[*CMP-2401*])
+
+* With this update, obsolete rules being applied to `s390x` have been removed. (link:https://issues.redhat.com/browse/CMP-2471[*CMP-2471*])
+
+[id="compliance-operator-1-4-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, for RHCOS systems using RHEL9, application of the `upstream-ocp4-kubelet-enable-protect-kernel-sysctl-file-exist` rule would fail. With this update, after auto remediation is applied, RHEL9-based RHCOS systems will show `PASS` upon the application of this rule. (link:https://issues.redhat.com/browse/OCPBUGS-13589[*OCPBUGS-13589*])
```

Comment:
[According to the SSG](https://redhat-documentation.github.io/supplementary-style-guide/#rhel), we shouldn't use RHEL, unless there's an exception specific to a team's usage. Is there an exception made for this feature?

Reply:
@bburt-rh good catch, and I worried about that. RHCOS-based containers come in "RHEL9" and "RHEL8" flavor only, and must be chosen according to that designation. We don't have a variable name for that, so I left it.

Reply:
The rule should be `ocp4-kubelet-enable-protect-kernel-sysctl-file-exist` for downstream build.
And the following description is not accurate: "With this update, after auto remediation is applied, RHEL9-based RHCOS systems will show `PASS` upon the application of this rule". We removed this rule, and use  `ocp4-kubelet-enable-protect-kernel-sysctl` instead.

Reply:
@xiaojiey I have made the requested change, thank you very much.